src/config.rs

```diff
@@ -23,6 +24,6 @@ mod test {
 
     #[test]
     fn test_reading_config() {
-        let cfg = dbg!(Config::load());
+        let _cfg = dbg!(Config::load());
     }
```

Comment:
Dlaczego podkreślasz tą zmienną?

Reply:
Jak dasz podkreślenie przed nazwą zmiennej to kompilator wie, że jej nie będziesz używał. Dzięki temu pozbywasz się warninga o unused zmiennej (albo errora - w przypadku Resultu, bo "result must be used")